core/claimmanager_test.go

```diff
@@ -16,6 +16,23 @@ import (
 	lpmscore "github.com/livepeer/lpms/core"
 )
 
+func TestShouldVerify(t *testing.T) {
```

Comment:
At some point, we should probably also write a test that compares the result of shouldVerifySegment with the expected results returned by the Solidity equivalent function just as a sanity check to confirm the Go implementation is producing the same number from the hashed inputs as Solidity that is used to mod with the verification rate